packages/odyssey-react/src/components/Checkbox/Checkbox.module.scss

```diff
@@ -155,40 +130,40 @@
 .label {
   display: block;
   position: relative;
-  margin-block-end: $spacing-xs;
-  padding-inline-start: 1em + $spacing-s-em;
-  font-size: $size-body-base;
-  font-weight: 400;
+  margin-block-end: var(--MarginBottom);
```

Comment:
are we using physical or logical values for our component theme variables? Right now we seem to be favoring logical cc @edburyenegren-okta

Reply:
This was my rationale as well. The language of our variables should match the language of our properties where reasonable.

While logical props are unfamiliar to most folks, this feels like an area we can quickly get people up to speed on.